src/sentry/templates/sentry/emails/unable-to-fetch-commits.html

```diff
@@ -4,6 +4,11 @@
   <h3>Unable to Fetch Commits</h3>
   <p>We were unable to fetch the commit log for your release (<em>{{ release.version }}</em>) due to the following error:</p>
   <p>{{ error_message }}</p>
+  <p><strong>Troubleshooting & References</strong></p>
```

Comment:
```suggestion
  <p><strong>Troubleshooting &amp; References</strong></p>
```

Need to escape HTML entities.